docs/source/redhat-opens-install.rst

```diff
@@ -53,27 +54,31 @@ Install OpenStack
 If you haven't already done so, install Openstack with Neutron and ML2 networking.
 Instructions for installing OpenStack on RHEL can be found here --
 `Juno <http://docs.openstack.org/juno/install-guide/install/yum/content/index.html>`__ /
-`Kilo <http://docs.openstack.org/kilo/install-guide/install/yum/content/index.html>`__.
+`Kilo <http://docs.openstack.org/kilo/install-guide/install/yum/content/index.html>`__ /
+`Liberty <http://docs.openstack.org/liberty/index.html>`__.
 
+At the time of writing the Liberty guide isn't yet available, but you can use
+PackStack instead: `https://www.rdoproject.org/QuickStart`__.
```

Comment:
Apparently Sphinx auto-detects links, so you don’t need the backticks or underscores around this link.